### ISSUE TITLE
 low accuracy

### ISSUE THREAD

**Issue by mkaskov** [Vict0rSch/deep_learning#4]
I tried your code. I think it has a very low accuracy.
I try to explain
1) You have to predict the next one value  (1 minute). Loss: 0.0854 - val_loss: 0.0721. The error is quite large. 
If you start predicting a few steps forward, based on previously predicted steps, it becomes obvious. 
![predict 50](https://cloud.githubusercontent.com/assets/14069473/16179446/2c716a76-3677-11e6-8bfd-7e0feb9ade62.png)
Have you tried to predict a few steps?
2) Using a simple and fast multi-layer Perceptron provides a significant increase in accuracy and performance, for example "dense" with 100-200-1. I test it for my task.

**Comment by vict0rsch** [comment]
The aim of the code is getting you to understand how to implement a network, codewise. I have to admit I haven't tried to optimize it and it is no surprise to me that a simpler model performs better, because I haven't tuned the parameters. 

My goal here was to get you started with an example network that is to be customized according to your own usage. 

PS : I haven't tried a few steps on this particular dataset + model, you are most welcome to propose a better model (that has to be recurrent to fit the task of the example ;) )

**Comment by jdelange** [comment]
Minor note: I get 

> The show accuracy argument is deprecated, instead you should pass the accuracy metric to the model at compile time: model.compile(optimizer, loss, metrics=["accuracy"])

**Comment by vgoklani** [comment]
@mkaskov What's the structure of your Perceptron? I'm curious to see if you've experimented with different activation functions and/or Dropout? Also, why did you choose 2 layers (presumably 100 and 200 hidden nodes, with a sigmoid activation function(?))
